client/my-sites/email/email-forwarding/email-forwarding-add-new-compact-list.tsx

```diff
@@ -96,7 +90,6 @@ const EmailForwardingAddNewCompactList = ( {
 				<Fragment key={ `email-forwarding__add-new_fragment__card-${ index }` }>
 					<div className="email-forwarding__add-new">
 						<EmailForwardingAddNewCompact
-							disabled={ isSubmittingEmailForward }
 							emailForwards={ [
```

Comment:
DId you forget to add the `isAddingEmailForward` toggle here to match the existing behaviour?